index.js

```diff
@@ -1,95 +1,124 @@
 // ***** Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1
+console.log("%cQuestion 1", "color: red") 
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+const name = "Kevin"
 
-// console.log(name)
-// // => "Ian"
-// name = "Not Ian" 
-// // => TypeError
-// console.log("%c----------", "color: red") 
+console.log(name)
+// => "Ian"
+//name = "Not Ian" 
+// => TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+ console.log("%cQuestion 3", "color: red") 
+ function drinkWater(thirst){
+   console.log( "Man I sure am thirsty")
+   thirst -= 1
+   console.log("Ahh that hits the spot")
+  return thirst
+ } 
 
-// console.log(drinkWater(12))
+
+ console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
 
-// console.log(sameSameButDifferent(5, 5)) 
+function sameSameButDifferent(num,maybeNum){
+  if (num === maybeNum)
+  return "same same"
+  else if (num == maybeNum)
+  return "same same (but different)"
+  else
+  return "different"
```

Comment:
This will work, but just a couple notes: 

- for if/else statements, it's better to use curly parentheses to separate out each condition for readability/flexibility:

```js
  if (num === maybeNum) {
    return "same same"
  } else if (num == maybeNum) {
    return "same same (but different)"
  } else {
    return "different"
  }
```

- Also, you can also convert the string to a number with `parseInt` to avoid `==`:

```js
} else if (num === parseInt(maybeNum)) {
```